### ISSUE TITLE
Icon in application indicator menu too big

### ISSUE THREAD

**Issue by syphernl** [dlech/Keebuntu#32]
I am using KeeAgent with KeePass on Ubuntu 15.10 with XFCE 4.12. 
However, the icon for KeeAgent is way too large in the menu when I right-click on the indicator icon:

![screenshot_2015-11-26_15-33-54](https://cloud.githubusercontent.com/assets/639906/11425168/413e14fc-9453-11e5-8c1d-d86b53175432.png)

Version details for the Keepass* packages:
```
keepass2 2.29+dfsg-1
keepass2-plugin-application-indicator 0.4.7+0.5.7
keepass2-plugin-keeagent 0.7.11
keepass2-plugin-keepasshttp 1.8.4
keepass2-plugin-libdbus 0.3.1+0.5.7
keepass2-plugin-libimagemagick 0.2.8+0.5.7
```

**Comment by dlech** [comment]
This usually means that imagemagick is not working. I'll have to check to see if the package name changed in 15.10. I should probably use a 16x16 icon to start with too.

**Comment by syphernl** [comment]
In regards to imagemagick, here are the currently installed version details on my 15.10 box:
```
ii  imagemagick                                                 8:6.8.9.9-5ubuntu2                         amd64        image manipulation programs -- binaries
ii  imagemagick-6.q16                                           8:6.8.9.9-5ubuntu2                         amd64        image manipulation programs -- quantum depth Q16
ii  imagemagick-common                                          8:6.8.9.9-5ubuntu2                         all          image manipulation programs -- infrastructure
```

**Comment by dlech** [comment]
The actual package I was referring to is `libmagickwand`. Looks like it got bumped to `libmagickwand-6.q16-2` in 15.10

**Comment by dlech** [comment]
Fix has been released in  keepass2-plugin-ubuntu - 0.6.0

It is available in `ppa:dlech/keepass2-plugins-beta` for xenial only.
